src/ArcCommander/ArcCommander.fsproj

```diff
@@ -5,6 +5,20 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
 	<GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
+  <PropertyGroup>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>ArcCommander</ToolCommandName>
+    <Authors>Heinrich Lukas Weil, Kevin Schneider, Timo Muehlhaus nfdi4plants OSS contributors</Authors>
+    <Description>Tool to manage your ARCs</Description>
+    <Summary>Tool to manage your ARCs</Summary>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/nfdi4plants/arcCommander</PackageProjectUrl>
+    <PackageTags>rdm dotnet arc</PackageTags>
+    <RepositoryUrl>https://github.com/nfdi4plants/arcCommander</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <FsDocsLicenseLink>https://github.com/nfdi4plants/arcCommander/blob/developer/LICENSE</FsDocsLicenseLink>
```

Comment:
I think `FsDocsLicenseLink` can be dropped as its deprecated

Reply:
https://fsprojects.github.io/FSharp.Formatting/styling.html#Customizing-via-Styling-Parameters